### PR TITLE
Move the FWB assets into the main cfgov assets

### PR DESCRIPTION
**Pull request by jimmynotjim**
The FWB assets were living on their own within the app code instead of
the main cfgov assets. Tidied up the code and utilized existing asset
code where possible. Could still use some more more optimization before
moving the scale code to Capital Framework as well as a discussion about
the flush well styles.

## Additions

- `o-well__flush` to pull a well left and right
- `o-well__stacked` to allow multiple wells to be stacked without spacing between them
- `o-scale` to control the default form element layouts within the wells
- `m-notification__subtle` to create less "in your face" notifications

## Testing

1. `gulp build`
1. Navigate to `http://localhost:8000/consumer-tools/financial-well-being/`, observe zero visible changes compared to `https://www.consumerfinance.gov/consumer-tools/financial-well-being/`

## Screenshots

screen shot 2017-11-09 at 9 39 40 am

## Notes

- I tried using combinations with `block` modifiers but due to spacing that was added in undesirable places I opted to use cfgov-refresh's `o-well`. Both `o-well` and `o-scale` should be moved to Capital Framework as a next step, but will probably depend on some HUB discussions.
- I didn't move the results styles because we don't yet have a great way to load assets for individual pages/components in Less. @Scotchester and I have discussed digging into that as a Design Ops project after we finish svg icons, @ascott1 what do you think?

## Todos

- Open/continue HUB discussions

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [x] Passes all existing automated tests
* [x] Any *change* in functionality is tested
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated
* [x] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/unprocessed/css/enhancements/notifications.less [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* CF Enhancements */
+.m-notification__subtle {
+    border-width: 0;
+    .m-notification_message {
+	    font-size: 1em;
+	}
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/enhancements/notifications.less]: Too far indented.
What does this notification look like? Is it absolutely necessary we have a variation of notifications?

Comment (jimmynotjim) [reply cfgov/unprocessed/css/enhancements/notifications.less]: Normal:

screen shot 2017-11-09 at 10 02 12 am

Subtle:

screen shot 2017-11-09 at 10 04 25 am

I know there's an ongoing HUB discussion about call out text, but I don't know where that's headed. This design is already live, just living in the FWB app markup.

Comment (anselmbradford) [reply cfgov/unprocessed/css/enhancements/notifications.less]: Okay the beta banner should probably use this too. Can you add a TODO at https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/molecules/global-banner.less#L13

Comment (jimmynotjim) [reply cfgov/unprocessed/css/enhancements/notifications.less]: Any changes there should go through that HUB discussion. I've added both of these examples to that issue.

File: cfgov/unprocessed/css/organisms/well.less [hunk 2]
```diff
@@ -70,6 +70,26 @@
...
+        border-right: none;
+
+        .respond-to-max( @bp-xs-max, {
+            border-left: none;
+            margin-right: -15px;
+            margin-left: -15px;
+        } );
+
+        .respond-to-min( @bp-sm-min, {
+            margin-left: -30px;
```
Comment (sebworks) [inline cfgov/unprocessed/css/organisms/well.less]: should these `em`?

Comment (jimmynotjim) [reply cfgov/unprocessed/css/organisms/well.less]: Ah yep, sorry, I'll fix those.

**Pull request merged**